src/sentry/static/sentry/app/components/search/sources/apiSource.jsx

```diff
@@ -90,7 +91,13 @@ async function createPluginResults(pluginsPromise, orgId) {
     })
     .map(plugin => ({
       title: plugin.isHidden ? `${plugin.name} (Legacy)` : plugin.name,
-      description: plugin.description,
+      description: (
+        <span
+          dangerouslySetInnerHTML={{
+            __html: markedSingleLine(plugin.description),
+          }}
```

Comment:
`markedSingleLine()` calls `dompurify.sanitize()` so this should be safe from xss

Reply:
Should we have multi-line and single line markdown components so that our `dangerouslySetInnerHTML` is contained to fewer places?

Reply:
@markstory @billyvg I am a bit confused. Is there something that I should change in my PR?

Reply:
I don't think you need to add it for this PR, but mark's suggestion is that we have components that can render markdown without having to have one-off `dangerouslySetInnerHTML`